### PR TITLE
fix: Client can't load in certain cases

### LINKED ISSUE #20231: Client can't load in certain cases
**Describe the bug**
Looks like the HTML5 client can't finish loading in some systems.. Reported in https://groups.google.com/g/bigbluebutton-dev/c/vQF1L0Zej54

**To Reproduce**
Not sure yet

**Expected behavior**
The client loads.

**Actual behavior**
The client is stuck while loading with error message (see screenshots)

**Screenshots**
Screenshot 2024-05-11 at 13 15 30
Screenshot 2024-05-13 at 09 30 40

**BBB version:**
BigBlueButton 3.0.0-alpha.6

### PR DESCRIPTION
**Pull request by ramonlsouza**
### What does this PR do?

Moves settings declaration to functions/components so they are not made on import time.

### Closes Issue(s)
Closes #20231

### CODE REVIEW

File: bigbluebutton-html5/imports/ui/components/video-provider/component.jsx [hunk 1]
```diff
@@ -241,13 +215,19 @@ class VideoProvider extends Component {
   }
 
   openWs() {
+    const {
+      connectionTimeout: WS_CONN_TIMEOUT = 4000,
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/video-provider/component.jsx:219]: Should we carry on the TODO we had (two spots)?

> // Default values and default empty object to be backwards compat with 2.2.
> // FIXME Remove hardcoded defaults 2.3.

File: bigbluebutton-html5/imports/ui/components/video-provider/service.js [hunk 2]
```diff
@@ -918,54 +953,11 @@ class VideoService {
     return constraints;
   }
 
-  applyCameraProfile (peer, profileId) {
-    const profile = CAMERA_PROFILES.find((targetProfile) => targetProfile.id === profileId);
-
-    // When this should be skipped:
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/video-provider/service.js:924]: The replacement portion seems more concise. Do we lose any functionality though? I am not spotting some of the proofchecks. Do we no longer need to log `video_provider_constraintchange_failed` ?

**Comment by antobinary** [comment]
Impressive rework!! :100:

File: bigbluebutton-html5/imports/ui/components/video-provider/service.js (hunk 2 above)
Comment (ramonlsouza) [reply bigbluebutton-html5/imports/ui/components/video-provider/service.js:924]: Some functions from video-provider service have been replaced by similar ones in graphql service file (see https://github.com/bigbluebutton/bigbluebutton/blob/v3.0.x-release/bigbluebutton-html5/imports/ui/components/video-provider/video-provider-graphql/service.ts#L422)

**Review by antobinary — APPROVED** [review]
In terms of code review and packaging changes LGTM!

**Review by antonbsa — APPROVED** [review]
I've tested the entire client and all the tests are passing, no bugs found! LGTM!

**Pull request merged**